CONTRIBUTING.md

```diff
@@ -32,6 +32,16 @@ The project's test suite can be run with the python test runner, `green`
 green -vvv  ./tests/unit
 ```
 
+In order to run the integration test, one will need to have a zowe profile. If zowe cli is installed, the test profile lives in `~/.zowe/profiles/zomsf/Zowe.yaml`
```

Comment:
The profile doesn't have to be called "Zowe.yaml", the name can be anything as long as it matches the `ZOWE_TEST_PROFILE` variable defined in your `.env` file.

Could you mention configuring the `.env` file here as well? To run the integration tests, it is necessary to define `ZOWE_TEST_PROFILE='<myProfile>'` in the `.env` file, so that tests will load the profile from `~/.zowe/profiles/zosmf/<myProfile>.yaml`.